src/models/People.js

```diff
@@ -85,7 +81,7 @@ static async displayPeopleProfessional() { // returns array of Timesheet Table
       ]);
 
       const capabilities = rawCapabilities.records.map(([{ stringValue: capability }])=> capability);
-      const roles = rawRoles.records.map(([{ stringValue: roles }])=> roles);
+      const roles = rawRoles.records.map(([{ stringValue: role }])=> role);
```

Comment:
This should change accordingly as well.
```suggestion
      const groups = rawGroups.records.map(([{ stringValue: group }])=> group);
```